ros2_control_demo_description/rrbot_description/ros2_control/external_rrbot_force_torque_sensor.ros2_control.xacro

```diff
@@ -18,7 +18,7 @@
         </xacro:unless>
       </hardware>
 
-      <sensor name="tcp_fts_sensor">
+      <sensor name="${prefix}tcp_fts_sensor">
```

Comment:
Nit: Shouldn't there be some delimiter (maybe an underscore) between `${prefix}` and the rest of the identifier? Not that it matters too much but I think I prefer `foo_tcp_fts_sensor` over `footcp_fts_sensor`.

Reply:
Prefixes aren't always used, in which case if this was changed you'd get something like `_tcp_fts_sensor`. When assigning a prefix the pattern I've seen here is to add the delimiter to the prefix. Something like: `foo_`. I agree that `foo_tcp_fts_sensor` is definitely better than `footcp_fts_sensor`.

Reply:
Good observation, in which case I'd suggest something like this:
```xml
<xacro:macro name="external_rrbot_force_torque_sensor" params="name prefix:='' use_sim:=^|false use_fake_hardware:=^|true fake_sensor_commands:=^|false">

...

<xacro:property name="prefix_" value='${prefix + "_" if prefix else ""}' />
<sensor name="${prefix_}tcp_fts_sensor">

...
```

Reply:
That is a nice way to handle that. I guess it's up to the owners if they would like to see that updated!

Reply:
@aprotyas this is a very good idea! And I was using this before. Nevertheless, I realized that having implicit changes is confusing for users. In the current way the `prefix` is very explicit, and it will be directly applied to the names. You can always as whatever delimiter you like in your launch files. 

Simple answer: "explicity before implicity" (IMHO it's easier to debug)

Reply:
I like that thought. Is there a way to document this behavior anywhere, maybe in the README?